src/templates/JobTemplate.jsx

```diff
@@ -42,18 +43,22 @@ const JobView = ({ data }) => {
             )}
           </div>
         </div>
-        <Card className="job-content">
+        <Card fluid className="job-content">
           <h1>{job.job_title}</h1>
-          <Markdown>{job.content.content}</Markdown>
+          <div class="description">
```

Comment:
Error in the console: `Warning: Invalid DOM property `class`. Did you mean `className`?`
Please check your pages for errors next time!